documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1080 @@
+<5> Enter the CA certificate for the API server of the OpenShift cluster, unless it was replaced by a third-party certificate, in which case, enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
+<6> Specify the API end point URL, for example, `https://<engine_host>/ovirt-engine/api`.
+endif::[]
+ifdef::ova[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: ova
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  url: <nfs_server:/nfs_path> <2>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> where: `nfs_server` is an IP or hostname of the server where the share was created and `nfs_path` is the path on the server where the OVA files are stored.
+endif::[]
+ifdef::ostack[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: openstack
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  domainName: <domain_name>
+  projectName: <project_name>
+  regionName: <region name> <5>
+  cacert: | <6>
+    <ca_certificate>
+  url: <api_end_point> <7>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the {osp} user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, specify `false` to verify the certificate. Defaults to `false` if not specified. Skipping certificate verification proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed.
+<5> Specify the name of the {osp} region.
+<6> Enter the CA certificate for connecting to the source environment. The certificate is not used when `insecureSkipVerify` is set to `<true>`. Note that if you set `insecureSkipVerify` to `false` and you do not set  `cacert`, {project-short} attempts to use the system CA.
```

Comment:
`The certificate is not used when insecureSkipVerify is set to <true>` - this also applies to the cacert property for other providers and we don't mention this above, do we really need this?

`Note that if you set insecureSkipVerify to false and you do not set  cacert, {project-short} attempts to use the system CA.` - this also applies to other providers and we don't mention it above, how about adding something like: `When this field is not set and 'skip certificate verification' is disabled, {project-short} attempts to use the system CA`, to all place where the cacert property appears?

Reply:
@RichardHoch start here.

Reply:
@ahadas I replaced all explanations of `cacert`, including vSphere and RHV, with "When this field is not set and 'skip certificate verification' is disabled, {project-short} attempts to use the system CA." Let me know if you wanted those two to be different, or if i otherwise misunderstood.